migrations/20220117025847_email_data.up.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TYPE valid_status AS ENUM ('pending', 'running', 'completed', 'stopped');
+CREATE TABLE blk_vrfy_job (
+    id SERIAL PRIMARY KEY,
+    job_uuid UUID,
+    created_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+    attempt_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+    total_records INTEGER NOT NULL,
+    total_processed INTEGER NOT NULL DEFAULT 0,
+    summary_total_safe INTEGER NOT NULL DEFAULT 0,
+    summary_total_invalid INTEGER NOT NULL DEFAULT 0,
+    summary_total_risky INTEGER NOT NULL DEFAULT 0,
+    summary_total_unknown INTEGER NOT NULL DEFAULT 0,
+    job_status VALID_STATUS NOT NULL DEFAULT VALID_STATUS 'pending'
```

Comment:
TBH, I think we don't need any of these fields in the DB. `total_records, total_processed` can be inferred from the sqlmxq status, and `summary_*` fields from `ema_vrfy_rec` if we store the email verification results there.

Reply:
Ok, this would mean that a get status call to a job will no longer be a record lookup but an aggregate query that would filter and count `ema_vrfy_rec` records to construct the response.

This might be an issue if the job status endpoint is accessed frequently.

Reply:
>  if the job status endpoint is accessed frequently.

I would prefer to focus on consistency and DB lean-ness first. And if what you say happens, we can always add a cache layer later for this aggregate results.

Reply:
Got it. The latest schema is oriented towards the goals you mentioned.